internal/app/issue/issue_handler.go

```diff
@@ -247,6 +248,9 @@ func (is *issueHandler) AddComponentVersionToIssue(issueId, componentVersionId i
 	err := is.database.AddComponentVersionToIssue(issueId, componentVersionId)
 
 	if err != nil {
+		if strings.HasPrefix(err.Error(), "Error 1062") {
```

Comment:
We should define some Database Errors that are returned by the database layer. Here we have now a dependency to the mariadb

Reply:
I agree